tasks/go.py

```diff
@@ -396,7 +397,20 @@ def check_mod_tidy(ctx, test_folder="testmodule"):
 
 @task
 def tidy_all(ctx):
-    for mod in DEFAULT_MODULES.values():
+    print(color_message('This command is deprecated, please use `tidy` instead', "orange"))
+    print("Running `tidy`...")
+    tidy(ctx)
+
+
+@task
+def tidy(ctx, only_modified_packages=False):
+    # TODO: if only_modified_packages then use `go mod tidy -diff` with golang 1.23
```

Comment:
The `-diff` arg will allow stopping early when we just want to check whether there was a change, at that point it might be clearer to make a separate `check-mod-tidy` invoke task

Reply:
Gotcha!